javascript/operators/array_comprehensions.json

```diff
@@ -0,0 +1,61 @@
+{
+  "javascript": {
+    "operators": {
+      "array_comprehensions": {
+        "__compat": {
+          "description": "Array comprehensions",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Array_comprehensions",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "30"
```

Comment:
The first note is no longer valid. The second talks about adding this non-standard thing to destructuring, which won't happen if there is no standard. I think we should leave this out of the data. It might be that all this will be removed (as it is non-standard) anyways.

Reply:
Sounds good for me.